vaas-app/src/vaas/purger/purger.py

```diff
@@ -12,23 +16,34 @@ def log_and_return_data(self, responses_summary):
         self.logger.debug(responses_summary)
         return responses_summary
 
-    def purge_url(self, url, servers):
+    def purge_url(self, url, servers, headers=None):
```

Comment:
We assumed that headers contains map <header_name: values_list> for example: 

```
'Accept': ['image/webp', 'image/jpeg']
'Accept-Language': ['pl-PL', 'en-US']
```

So for above input should be prepared purge executions for each combination as below:

### 1.
```
Accept: image/webp
Accept-Language: pl-PL
```

### 2.
```
Accept: image/webp
Accept-Language: en-US
```

### 3.
```
Accept: image/jpeg
Accept-Language: pl-PL
```

### 4.
```
Accept: image/jpeg
Accept-Language: en-US
```

Reply:
Added this functionality in next commits.